torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py

```diff
@@ -445,18 +452,24 @@ def _annotate_conv2d(
         if _is_annotated([conv_node]):
             return
 
-        input_node = conv_node.args[0]
-        weight_node = conv_node.args[1]
-        bias_node = conv_node.args[2]
-        conv_node.meta["target_dtype_info"] = {
-            "input_act_obs_or_fq_ctr_map": {
-                input_node: get_act_obs_or_fq_ctr(quantization_config),
-                weight_node: get_weight_obs_or_fq_ctr(quantization_config),
-                bias_node: get_bias_obs_or_fq_ctr(quantization_config),
+        input_act = conv_node.args[0]
+        assert isinstance(input_act, Node)
+        weight = conv_node.args[1]
+        assert isinstance(weight, Node)
+        bias = conv_node.args[2]
+        assert isinstance(bias, Node)
```

Comment:
Maybe we can consider to add a function such as `_is_all_fx_node(node_list)`? It can reduce the lines of duplicated code.

Reply:
this annotation code will be changed in the future to use module_partition API, and this may not be needed

Reply:
Got it. Thanks.